server/routes/requests.js

```diff
@@ -322,4 +323,53 @@ router.get('/getConfirmed/:id/:timeOffset', passport.authenticate('jwt', {sessio
     }
 })
 
+router.patch('/actionItems/:id/:timeOffset', passport.authenticate('jwt', {session: false}), async (req, res, next) => {
+    let alumniId = req.params.id;
+    let timeOffset = req.params.timeOffset;
+    let requestId = req.body.requestId;
+    let actionItems = req.body.actionItems;
+    let conditions = ['Awaiting Confirmation', 'Confirmed', 'Completed']
+    let requests = []
+    try {
+        let request = await requestSchema.findById(requestId);
+        request.actionItems = actionItems;
+        await request.save()
+        for (let status of conditions) {
+            const dbData = await requestSchema.find({mentor: alumniId, status: status}).populate('student')
+            for (let request of dbData) {
+                request.time = timezoneHelpers.applyTimezone(request.time, timeOffset)
+            }
+            requests.push(dbData)
+        }
+        res.json({'requests' : requests});
+    } catch (e) {
+        console.log('/request/actionItems Error: ' + e)
+        res.status(500).send({
+            message: 'Failed to add alumni note: ' + e
+        })
+    }
+
+
+});
+
+router.post('/addActionItem/', async (req, res, next) => {
```

Comment:
I understand the context for the conversation from earlier now! We can remove this end-point, and have any new actionItems be created in the PATCH `/actionItems/:id` request above.
An example of how new actionItems and existing actionItems can be passed in req.body can be found in `alumni.js`'s `/interests/add/:id` PATCH end-point.